WcaOnRails/app/assets/stylesheets/polls.scss

```diff
@@ -0,0 +1,39 @@
+#poll-results {
+  width: auto;
+  td {
+    padding-left: 20px;
+
+    &.total {
+      font-weight: bold;
+    }
+
+    &.bar {
+      width: 200px;
+    }
+  }
+
+
+}
+
+.polls-list {
+  width: auto;
+}
+
+.nested-fields input {
+  width: 300px;
+  display: inline;
+}
+
+a.new-poll {
+  color: #090;
+}
+
+ẗr.voted {
+  background-color: #4f5;
+}
+
+.polls-home {
```

Comment:
What?
I have this link in several places, at the bottom of the page. You mean moving it up on all of them?